src/app-layout/visual-refresh/drawers.tsx

```diff
@@ -161,6 +206,63 @@ function DesktopTriggers() {
   const hasSplitPanel = splitPanel && splitPanelDisplayed && splitPanelPosition === 'side' ? true : false;
   const previousActiveDrawerId = useRef(activeDrawerId);
 
+  const triggersContainerRef = useRef<HTMLDivElement>(null);
+
+  const splitPanelHeight =
+    isSplitPanelOpen && splitPanelPosition === 'bottom' ? splitPanelReportedSize : splitPanelReportedHeaderHeight;
+  const containerHeight = useContainerHeight(triggersContainerRef) || triggersContainerRef.current?.clientHeight;
+
+  const getIndexOfOverflowItem = () => {
+    if (containerHeight) {
+      const ITEM_HEIGHT = 48;
+      const overflowSpot =
+        activeDrawerId && isSplitPanelOpen
+          ? (containerHeight - splitPanelReportedHeaderHeight) / 1.5
```

Comment:
<img width="1862" alt="image" src="https://github.com/cloudscape-design/components/assets/9701338/d61e18c4-d86f-4b06-86a4-b70583d32da7">
I'm not sure we can - based on the grid lines here, there aren't any that correspond to the top of the split panel. Unless I'm missing something?

Reply:
Checked the implementation one more time and you are correct. Split panel sticks to the bottom using `align-self`, no easy wins here. I am okay with the current code with this discovery.

Reply:
However when inspecting this setup, I found a corner case

Here we want the overflow to consider split panel height:

<img width="742" alt="Screenshot 2023-08-11 at 16 44 45" src="https://github.com/cloudscape-design/components/assets/812240/34027b5a-3183-4e62-8081-0aa2a2c7d6c5">

But in this one we probably do not (because drawers bar has the full height):

<img width="735" alt="Screenshot 2023-08-11 at 16 44 58" src="https://github.com/cloudscape-design/components/assets/812240/82496cff-0629-48c9-a5d9-fec51f5b7b69">

What do you think? Is this important enough to address?

---

🤔  When trying to address this, the height calculation condition becomes even more convoluted. The idea from this comment may become more valuable https://github.com/cloudscape-design/components/pull/1416#discussion_r1288281996, because it separates tasks "make buttons overflow if they do not fit container" and "adjust container height to not overlap split panel". Two simpler tasks are usually more preferred than a single complex one.

Reply:
This calculation already takes this into account (see the `activeDrawerId &&` logic in the code).

Reply:
It seems like there some fixes were made. I cannot reproduce the previous behavior on the latest version.

resolving the thread.